src/sources/notification_permissions.ts

```diff
@@ -1,7 +1,16 @@
-import { BotdError, State } from '../types'
+import { BotdError, BrowserEngineKind, BrowserKind, State } from '../types'
+import { getBrowserEngineKind, getBrowserKind, isDesktopWebKit } from '../utils/browser'
 
 export default async function getNotificationPermissions(): Promise<boolean> {
   if (window.Notification === undefined) {
+    // if on mobile ios, ignore
```

Comment:
I"m not sure if this is what we want to do. The `BotdError` was expected to be thrown when `window.Notification` is undefined, why do we change the behaviour now for Safari? Shouldn't we just update the tests to expect the `BotdError` in Safari 17?

Reply:
I removed the check